src/app.jsx

```diff
@@ -528,6 +539,37 @@ class Application extends React.Component {
                     this.setState({ userLingeringEnabled: true });
                 }
             });
+
+            // detect which other users have containers running
+            cockpit.spawn([
+                'find', '/sys/fs/cgroup',
+                // RHEL 8 version still calls it "podman-*.scope", newer ones "libpod*"
+                '-name', 'libpod*s.cope', '-o', '-name', 'podman-*.scope',
+                '-exec', 'stat', '--format=%u %U', '{}', ';'],
+                          { superuser: "require", error: "message" })
```

Comment:
Require seems unneeded, as I can list root's cgroup as normal user:

```
find /sys/fs/cgroup -iname  'libpod*s.cope' -o -name 'podman-*.scope' -exec stat --format="%u %U" {} \;
0 root
1000 jelle
```

Also this does not seem to work with quadlets, I guess since they are under a different cgroup scope? Not sure if we care about that? It seems we need to detect something like this then:

```
/sys/fs/cgroup/system.slice/jellyfin.service/libpod-payload-43d14c04471cef173a77ce5ffab61cfd793a31e833b916aeeb36a2bf549c26d5/
```

Reply:
> Require seems unneeded

It's a simple/sneaky way to avoid the detection altogether if you don't have root, as *connecting* to other user's podman services definitively needs root. I'll add a comment.

As for quadlets, I'll add an integration test and fix it up, thanks!

Reply:
Curiously quadlets work as there is a (hidden) podman-pause container, or at least cgroup. But let's not rely on this, but detect it from sysfs directly.

Reply:
Done!